Curs04/src/views/evaluation.js

```diff
@@ -0,0 +1,111 @@
+function EvaluationsPage (options = {}) {
+
+    const headings = [
+        'Nume','Tehnologie','Nivel'
+    ]
+
+    const rows = [
+
+        {
+            Name: "Popescu Adrian",
+            Technology: "Javascript",
+            Level: "Mid 2"
+        },
+        {
+            Name: "Dragan Roxana",
+            Technology: "PHP",
+            Level: "Mid 1"
+        },
+        {
+            Name: "Florescu Mihai",
+            Technology: "Javascript",
+            Level: "Junior 3"
+        },
+        {
+            Name: "Gheorghe Andrei",
+            Technology: "Ruby",
+            Level: "Senior 1"
+        }
+    ]
+
+    return `
+
+        ${NAV()}
+        ${EvaluationsTable({
+            items: rows,
+            itemHeadings: headings
+        })}
+        ${Footer()}
+         `;
+}
+
+function EvaluationsTableHeader(options) {
+
+    const headings = [];
+
+    options.headings.forEach(function(el) {
```

Comment:
So you have a sequence that generates an array containing elements based on the elements of another array. Can you think of any other way to achieve this other than using forEach?

Reply:
Yes. We can use the map function and we can make an array with the header options.

With the map function we can do into the array and to display every option.